securedrop/manage.py

```diff
@@ -247,10 +256,15 @@ def get_args():
     return parser
 
 
+def setup_verbosity(args):
+    logging.getLogger(__name__).setLevel(args.verbose)
```

Comment:
I don't like how you're overloading `args.verbose`, which, as a flag, is implied to be a boolean, with constants indicating the log level. I think it is better to decouple "what the user asked for" from "how the user's request should be implemented internally."